### PR TITLE
replace social media widgets with icons that link to the social media…

### PR DESCRIPTION
**Pull request by tanyaparker**
… account

**updated markdown guide**
image

**/educate/community**
image

**/about**
image

**/about/2014**
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
This gets rid of most of them. Still on the backlog to work on the more complex widgets like on /congrats.

**Comment by joshlory** [comment]
Does it make sense to make these a partial?

**Comment by tanyaparker** [comment]
@davidsbailey fyi for content pipeline. Dave, do you have a preference for using a partial or not? Or any other thoughts on this PR?

I think a partial would be a little over engineering and I favor the flexibility to

1) change the order of appearance (maybe by relevance or importance)
2) chose a subset of social media accounts (otherwise I'd need to support a param to hide/show each platform)
3) link to different accounts like the teacher twitter account instead of general twitter account (I could start with the Code.org as default and allow a param to overwrite the target address)
4) link to multiple accounts like both the Code.org and Teacher twitter accounts

I think creating a partial would be overkill for these cases. Content editors should be able to use the template in the /markdown guide and adjust accordingly.

**Comment by davidsbailey** [comment]
Thanks for the heads up, Tanya! This change looks good to me for now.

In the future, we will need to get html out of markdown. at that time, extracting html into haml partials will be one of a few primary strategies. To make it so that content editors can customize the ordering, each social link would have to be its own partial. 

I would say, extracting a partial for these social links would be optional in this PR since isolating markdown from html is part of a separate stream of work. However, if you wanted to do that now it would definitely take us in a good direction and would be a great pattern for people to copy.

**Comment by tanyaparker** [comment]
Did some more restructuring on Hadi's bio as well.

Before | After
------------ | -------------
image | image

**Comment by tanyaparker** [comment]
Updated with a partial. Ready for review. :)

File: pegasus/sites.v3/code.org/public/markdown/more.md [hunk 1]
```diff
@@ -89,6 +89,13 @@ Cu vis minim imperdiet. Vel cu phaedrum adolescens, vim te assentior delicatissi
 
 
 ## Social Media
+
+### How to point to Code.org's social media sites
+Do not use any Facebook, Twitter, etc. widgets that connect users to like or follow Code.org on social media. These widgets use cookies, which violates our privacy policy. Instead, you can show an icon or image of the social media platform's logo and link to Code.org's social media accounts.
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/markdown/more.md]: Might be worth explaining that the difference is that we don't want to embed any content hosted on a third party site.  If we include an icon or image from that third-party site they'll also be able to track.

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/markdown/more.md]: What do you mean include an icon or image from another site? Anything other than FontAwesome?

Comment (breville) [reply pegasus/sites.v3/code.org/public/markdown/more.md]: Any time we include an asset from another site, it gives them a chance to track that user's activity.  So in this case the message is that we want to host the social media platform's logo/button on our own site.  We don't want to embed anything that we are not hosting when it comes to these social media assets.

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/markdown/more.md]: This more clear?








`Do not use any Facebook, Twitter, etc. widgets that connect users to like or follow Code.org on social media. These widgets use cookies that track our users, which violates our privacy policy. Instead, you can show an icon or image of the social media platform's logo and link to Code.org's social media accounts. By using this method, we host the social media platform's logo/button on our own site.`

**Pull request merged**